olm-catalog/serverless-operator/project.yaml

```diff
@@ -0,0 +1,29 @@
+---
+project:
+  name: serverless-operator
+  version: 1.11.0
+
+olm:
+  replaces: 1.10.0
+  skipRange: '>=1.10.0 <1.11.0'
+  channels:
+    default: '4.3'
+    list:
+      - '4.5'
+      - '4.6'
+
+requirements:
+  kube:
+    minVersion: 1.15.0
```

Comment:
Nit: Some versions are quoted, some are note. E.g. `replaces: 1.10.0` vs `default: '4.3'` Is that intentional?

Reply:
Yup. Quotes are values that need to be. Otherwise, they will be treated as numbers. `1.10.0` is a string, `'4.3'` need to be quoted to not be a number.

Reply:
I see. OK